### PR TITLE
Partitioner: do not offer RAID partitions to create another RAID

### PR DESCRIPTION
**Pull request by ancorgs**
Refinement over https://trello.com/c/G8Or8z0z/211-3-full-disks-as-member-of-md since we refined the requirements a little bit more after some chat with @aschnell

### CODE REVIEW

File: src/lib/y2partitioner/actions/controllers/md.rb [hunk 1]
```diff
@@ -287,7 +287,9 @@ def available_disks
         # @param partition [Y2Storage::Partition]
         # @return [Boolean] true if it is valid; false otherwise.
         def valid_for_md?(partition)
-          partition.id.is?(:linux_system) && !partition.type.is?(:extended)
+          partition.id.is?(:linux_system) &&
+            !partition.type.is?(:extended) &&
+            !partition.partitionable.is?(:raid)
```
Comment (jreidinger) [inline src/lib/y2partitioner/actions/controllers/md.rb]: method documentation does not mention that it filter out raid partitions. I also would like to see here in code rationale why we filter it out, as we easily forgot WHY in few months.

**Comment by ancorgs** [comment]
The commit updating the version/changelog was there... in my local copy ;-) I had forgotten to push it.

In addition, I improved documentation as requested.

**Comment by lslezak** [comment]
:heavy_check_mark: Internal Jenkins [job #31](https://ci.suse.de/job/yast-yast-storage-ng-master/31/) successfully finished
:heavy_check_mark: Created IBS submit [request #171739](https://build.suse.de/request/show/171739)

**Comment by lslezak** [comment]
:heavy_check_mark: Public Jenkins [job #29](http://ci.opensuse.org/job/yast-yast-storage-ng-master/29/) successfully finished
:heavy_check_mark: Created OBS submit [request #634719](https://build.opensuse.org/request/show/634719)

**Pull request merged**